server/config.ts

```diff
@@ -122,6 +118,7 @@ export default {
     },
     coordinatorApi: {
       url: get('COORDINATOR_API_URL', 'https://arns-coordinator-api-dev.hmpps.service.justice.gov.uk'),
+      healthPath: '/health/ping',
```

Comment:
Turned all of the above off in docker, but this one never reports DOWN...

Reply:
Did you turn off the wiremock docker container too?